overlord/snapstate/snapstate.go

```diff
@@ -80,6 +83,11 @@ func isParallelInstallable(snapsup *SnapSetup) error {
 	return fmt.Errorf("cannot install snap of type %v as %q", snapsup.Type, snapsup.InstanceName())
 }
 
+func requiredSpaceWithMargin(minSize uint64) uint64 {
+	// 10% extra + 1Mb
+	return minSize + uint64(0.1*float64(minSize)) + 1024*1024
```

Comment:
Integer division `minSize/10` could work here without casting to floating point.

Reply:
Thanks, I think it's a sensible suggestion.

Reply:
Done.